pom.xml

```
@@ -475,16 +475,36 @@
                         </goals>
                         <configuration>
                             <sourcePaths>
-                                <sourcePath>${basedir}/src/main/resources/template-schemas/ko.json</sourcePath>
+                                <sourcePath>${basedir}/src/main/resources/schemas/templates/ko.json</sourcePath>
                             </sourcePaths>
                             <targetPackage>it.pagopa.generated.notifications.templates.ko</targetPackage>
                             <classNameSuffix>Template</classNameSuffix>
                             <includeConstructors>true</includeConstructors>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>event-schemas</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <sourcePaths>
+                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1/event_version.json</sourcePath>
+                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1/transaction_activated_event.json</sourcePath>
+                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1/transaction_activation_requested_event.json</sourcePath>
+                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1/transaction_authorization_requested_event.json</sourcePath>
+                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1/transaction_authorization_status_updated_event.json</sourcePath>
+                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1/transaction_closure_sent_event.json</sourcePath>
+                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1/transaction_user_receipt_added_event.json</sourcePath>
+                            </sourcePaths>
```

```suggestion
                            <sourcePaths>
                                <sourcePath>${basedir}/src/main/resources/schemas/events/v1</sourcePath>
                            </sourcePaths>
```
You can specify the folder containing all schemas so that you don't have to add newly added schemas in the future here